pkg/test/util.go

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  *
  */
+
+// Package test is defined skip the specific test if we are not use running in integration testing mode
```

Comment:
I would make this comment more generic, something like: `Package test contains specific test utilities used throughout the operator`